test/unit/math/prim/scal/fun/binomial_coefficient_log_test.cpp

```diff
@@ -27,7 +28,14 @@ TEST(MathFunctions, binomial_coefficient_log) {
   }
 
   test_binom_coefficient(1e9, 1e5);
-  test_binom_coefficient(1e50, 1e45);
+
+  // NOTE(SW): 2019-05-31 replaced large number test as
```

Comment:
Whoa. This wasn't listed in the side effects. If I understand this correctly, your PR changes the behavior for this function with large values?

What happens when you run the test with `1e50, 1e45`? What was it before and what is it now?

Reply:
The large number test did always overflow (so it still does on develop). The std lgamma returned Inf while the boost returns NaN. I need to check the lgamma_r, but I would guess it returns Inf (the lgamma_r only differs in how the sign is stored of the argument). I changed the test so that we avoid the overflow and actually do a meaningful test here.